src/workerd/api/queue.c++

```diff
@@ -386,23 +382,29 @@ void QueueMessage::retry() {
     IoContext::current().logWarning(msg);
     return;
   }
-  result->explicitRetries.findOrCreate(id, [this]() { return kj::heapString(id); } );
```

Comment:
Don't we need to insert results into both `explicitRetries` and the new `retries` field until the consumers have this data have been switched over to use the new field? Unless you've already updated the queues code to read both fields and merge the results, this looks like it could cause bugs during the transition.

Reply:
Correct, queue code is already consolidating/merging both fields. In any case, I planning to remove deprecated fields in this PR.